### PR TITLE
add home page (index.html)

### PR DESCRIPTION
**Pull request by Filimon523**
Hey guys, I created this home page(index.html). Please review my work. Thanks a lot , wish you a blessed day.

### CODE REVIEW

**Review by RhstiSWE — COMMENTED** [review]
Good  job File  i like your work keep it up.

**Review by Filimon523 — COMMENTED** [review]
Thanks alot.

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                </div>
+                <div class="footer-section">
+                    <h3>Contact Us</h3>
+                    <p>How to reach us.</p>
+                </div>
+            </div>
+            <p>Copyright &copy; 2024 Amazon&trade;</p>
+        </footer>
+    </div>
+</body>
```
Comment (RhstiSWE) [inline src/index.html:85]: file before you finish it you must add products html.

Comment (Filimon523) [reply src/index.html:85]: okay the product.html page, hermon will finish it.

**Review by Filimon523 — COMMENTED** [review]
For test purpose..

**Pull request merged**